_example/hello-world/Makefile

```diff
@@ -10,16 +10,16 @@ pack:
 
 dbuild:
 	@docker run --rm \
-		-u ${shell id -u}:${shell id -g} \
```

Comment:
This command has been added to prevent the creation of outputs files in the ownership of "root". If it does not work on OSX, we should find an alternative.
Here is what I suggest, please try this one on OSX and if it is ok then submit changes accordingly:
```
	-u `stat -c "%u:%g" .` \
```